cli/lib/platforms.js

```diff
@@ -0,0 +1,71 @@
+const fs = require("fs");
+const path = require("path");
+const Parcel = require("parcel-bundler");
+
+const web = {
+  async build(destination, skipBundle) {
+    if (skipBundle) return;
+    const bundler = await setupParcel(destination);
+    await bundler.bundle();
+  },
+  async run(path) {
+    const bundler = await setupParcel(path, { watch: true });
+    await bundler.serve();
+  }
+};
+
+const node = {
+  async build() {},
```

Comment:
This looks like the node target does not have to be built at all🤔 Maybe we can move the build logic in here? Possibly, we can have a `generic` build target that just outputs js, which will then be used by the node and the web target.